### PR TITLE
Adding file ext var

### PR DESCRIPTION
**Pull request by vanhtuan0409**
Some formatters support multiple file format but requires file extension hint (eg: `deno fmt`).

Adding `FILEEXT` var might help with those kind of formatter.

### CODE REVIEW

**Comment by vanhtuan0409** [comment]
Hi @mattn , could you take a look at this PR? Sorry that I didn't discuss about this PR before hand

**Comment by disrupted** [comment]
this is exactly what I was looking for couple of days ago. Thanks for adding it!

File: langserver/handler.go [hunk 1]
```diff
@@ -589,7 +590,10 @@ func (h *langHandler) handle(ctx context.Context, conn *jsonrpc2.Conn, req *json
 }
 
 func replaceCommandInputFilename(command string, fname string) string {
+	ext := path.Ext(fname)
```
Comment (mattn) [inline langserver/handler.go:594]: Why you don't use filepath.Ext ?

Comment (vanhtuan0409) [reply langserver/handler.go:594]: I didn't aware of the `filepath.Ext` function. Do you want me to change it over?

Comment (vanhtuan0409) [reply langserver/handler.go]: changed to filepath package instead

Comment (vanhtuan0409) [reply langserver/handler.go]: Hi @mattn , can you check it out again?

**Comment by disrupted** [comment]
Hi @mattn! Did you have the chance to take another look at this? LGTM imo.

**Comment by mattn** [comment]
Thank you

**Pull request merged**